src/Core/Application/Catalog/Services/BrandService.cs

```diff
@@ -65,7 +65,7 @@ public async Task<Result<Guid>> UpdateBrandAsync(UpdateBrandRequest request, Gui
         if (brand == null) throw new EntityNotFoundException(string.Format(_localizer["brand.notfound"], id));
         var updatedBrand = brand.Update(request.Name, request.Description);
         updatedBrand.DomainEvents.Add(new StatsChangedEvent());
-        await _repository.UpdateAsync(updatedBrand);
+        await _repository.UpdateAsync<Brand>(updatedBrand);
```

Comment:
This and all the other places where types are mentioned shouldn't be necessary anymore with the default, right?

Reply:
tnx, my bad